### DISCUSSION TITLE
Deploying Sql Server Always On Availability Group on Kubernetes

### DISCUSSION THREAD

**Discussion by giscus** [PradeepLoganathan/pradeepl-blog#25] [BlogPosts]
# Deploying Sql Server Always On Availability Group on Kubernetes

In this post we will deploy MS Sql Server in a kubernetes cluster. We will then configure it as an always on availability group.We will also look at synchronization and failover.

https://pradeepl.com/blog/kubernetes/deploying-sql-server-on-kubernetes/

**Comment by sadeghsobhi** [comment]
hi please help me
-- Create a master key and certificate
CREATE MASTER KEY ENCRYPTION BY PASSWORD = 'Welcome@0001234567';
GO
CREATE CERTIFICATE ag_certificate WITH SUBJECT = 'ag_certificate';
when run this T-SQL code error access deny

**Comment by mgaldamez** [comment]
Hi, first of all, thank you for this amazing tutorial.
now the question jeje :)
what happend if im in failover mode (my secondary1 replica is my primary replica) can i switch back to the primary? 
or failover again to primary replica?
